sdk/formrecognizer/azure-ai-formrecognizer/samples/README.md

```diff
@@ -63,6 +64,7 @@ All of these samples need the endpoint to your Form Recognizer resource ([instru
 |[sample_get_bounding_boxes.py][sample_get_bounding_boxes] and [sample_get_bounding_boxes_async.py][sample_get_bounding_boxes_async]|Get info to visualize the outlines of form content and fields, which can be used for manual validation|
 |[sample_differentiate_output_models_trained_with_and_without_labels.py][sample_differentiate_output_models_trained_with_and_without_labels] and [sample_differentiate_output_models_trained_with_and_without_labels_async.py][sample_differentiate_output_models_trained_with_and_without_labels_async]|See the differences in output when using a custom model trained with labeled data and one trained with unlabeled data|
 |[sample_differentiate_output_labeled_tables.py][sample_differentiate_output_labeled_tables] and [sample_differentiate_output_labeled_tables_async.py][sample_differentiate_output_labeled_tables_async]|See the differences in output when using a custom model trained with fixed vs. dynamic table tags|
+|[sample_convert_to_and_from_dict.py][sample_convert_to_and_from_dict_v3_1] and [sample_convert_to_and_from_dict_async.py][sample_convert_to_and_from_dict_async_v3_1]|Convert convenience models to a dictionary that can be used to create JSON content, then convert the same dictionary back to the original model class|
```

Comment:
nit: not sure we refer to them as "convenience models" anywhere else. Maybe just "model types"?

Reply:
Ok sounds good...just didnt want people to confuse these models with the generated ones...but agree dont want to introduce a strange new concept for customers. Done!